### PR TITLE
Loan Request validation/closure route

### LINKED ISSUE #9: Define API routes to validate/close LoanRequest

### PR DESCRIPTION
**Pull request by ricardomilhazes**
- Added methods for validation and closure of Loan Requests

### CODE REVIEW

File: tuichain/api/views/loanrequests.py [hunk 1]
```diff
@@ -40,6 +40,46 @@ def create_loan_request(request):
 
     return Response({'message': 'Loan Request successfully created'},status=HTTP_201_CREATED)
 
+@api_view(["POST"])
```
Comment (hpereira98) [inline tuichain/api/views/loanrequests.py]: the request is an update, so the method should be a PUT

File: tuichain/api/views/loanrequests.py [hunk 2]
```diff
@@ -40,6 +40,46 @@ def create_loan_request(request):
...
+
+    if loanrequest.validated:
+        return Response({'error': 'The given Loan Request as already been validated'},status=HTTP_403_FORBIDDEN)
+
+    loanrequest.validated = True
+    loanrequest.save()
+
+    return Response({'message': 'Loan Request has been validated'}, status=HTTP_201_CREATED)
+
+@api_view(["POST"])
```
Comment (hpereira98) [inline tuichain/api/views/loanrequests.py]: same as above

**Review by hpereira98 — CHANGES_REQUESTED** [review]
Besides the changes I pointed out, everything seems fine. Nice job!

File: tuichain/api/views/loanrequests.py (hunk 2 above)
Comment (ricardomilhazes) [reply tuichain/api/views/loanrequests.py]: Yeah, good point ahah my bad

**Review by hpereira98 — APPROVED** [review]

**Review by mrr37 — COMMENTED** [review]
'as' -> 'has' don't know if these corrections are important (56, 76)

**Review by ricardofsc10 — APPROVED** [review]
Nothing but some grammar errors as @mrr37 pointed out

**Pull request merged**